### ISSUE TITLE
trained model (train_unet_demo) gives bad images.h5

### ISSUE THREAD

**Issue by marinapapageorgiou** [facebookresearch/fastMRI#349]
Hello @mmuckley,
I am studing singlecoil (Unet model) and knee.

I run the pretrained model (run_pretrained_unet_inference.py) and the results are these images:
![1](https://github.com/user-attachments/assets/a59f6108-5fa1-4564-8fa7-f6d16240bdb0)
![2](https://github.com/user-attachments/assets/a5af5a25-6f13-496e-9fab-da169100726c)

After I trained the Unet model (train_unet_demo.py) and I tested it. The images of the testing are these:
![3](https://github.com/user-attachments/assets/d54bd59f-dde7-4d2d-baa5-980fd35483ea)
![4](https://github.com/user-attachments/assets/151ac2e7-984b-431a-9f79-e3b98f1ea4bd)

If it is usuful I run the model with max_epochs=50 and lr=0.001, # learning rate. And i tried also with  max_epochs=50 and lr=0.0003, # learning rate. But in both cases the best epoch that is keeps as the best is epoch=41.
Also, I am using only these files for dataset
knee_singlecoil_train
knee_singlecoil_val
knee_singlecoil_test

So my question is, What is wrong with the testing images of my training model? Is the dataset not big enough? Is the learning rate not appropriate? Is something else? Does anyone know?
